### DISCUSSION TITLE
OP3 Prefix Not Found

### DISCUSSION THREAD

**Discussion by britaliano** [skymethod/op3#30] [Q&A]
Hello everyone, I have just set up a new podcast, the first episode has been posted and the RSS is here:

https://fblive.yourpodcast.agency/feed/podcast/football-live/

Whenever I try to Set Up on op3.dev, I get the OP3 Prefix not found message. It also says No Episodes.

The URL of the first episode is https://op3.dev/e/fblive.yourpodcast.agency/podcast-download/17/football-live-podcast-20-21-april-2024.mp3?ref=download

I have clearly done something wrong, can someone steer me in the right direction?

I am self-hosting the podcast using Wordpress and the Seriously Simple Podcasting plugin.

**Comment by johnspurlock-skymethod** [comment]
Hi there, it doesn't look like your first episode is actually in your rss feed yet

currently, https://fblive.yourpodcast.agency/feed/podcast/football-live/ has no episodes

<img width="1141" alt="image" src="https://github.com/skymethod/op3/assets/47259736/95ce6a54-6db4-4aad-86ab-dac990faa2dd">

if no episodes are in the feed, there is nothing for podcast players to play! : )

**Comment by britaliano** [comment]
That does make sense, but here is the RSS Feed as I see it:

`<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="https://fblive.yourpodcast.agency/wp-content/plugins/seriously-simple-podcasting/templates/feed-stylesheet.xsl"?><rss version="2.0"
	 xmlns:content="http://purl.org/rss/1.0/modules/content/"
	 xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	 xmlns:dc="http://purl.org/dc/elements/1.1/"
	 xmlns:atom="http://www.w3.org/2005/Atom"
	 xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	 xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	 xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"
	 xmlns:googleplay="http://www.google.com/schemas/play-podcasts/1.0"
	 xmlns:podcast="https://podcastindex.org/namespace/1.0"
	>
		<channel>
		<title>The Football Live Podcast</title>
		<atom:link href="[https://fblive.yourpodcast.agency/feed/podcast/football-live/](view-source:https://fblive.yourpodcast.agency/feed/podcast/football-live/)" rel="self" type="application/rss+xml"/>
		<link>https://fblive.yourpodcast.agency/series/football-live/</link>
		<description>Football Live is exclusive to Brila FM, providing top class, live coverage of the major leagues in Europe to the people of Nigeria. Our show is regularly listened to by audiences on several million in Nigeria but it is available globally for you to listen to live.</description>
		<lastBuildDate>Thu, 25 Apr 2024 17:05:05 +0000</lastBuildDate>
		<language>en-US</language>
		<copyright>© Football Live</copyright>
		<itunes:subtitle>The biggest football show on Earth!</itunes:subtitle>
		<itunes:author>Football Live</itunes:author>
		<itunes:summary>Football Live is exclusive to Brila FM, providing top class, live coverage of the major leagues in Europe to the people of Nigeria. Our show is regularly listened to by audiences on several million in Nigeria but it is available globally for you to listen to live.</itunes:summary>
		<itunes:owner>
			<itunes:name>Football Live</itunes:name>
			<itunes:email>buono@buono.net</itunes:email>
		</itunes:owner>
		<itunes:explicit>false</itunes:explicit>
		<itunes:image href="[http://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg](view-source:http://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg)"></itunes:image>
			<image>
				<url>http://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg</url>
				<title>The Football Live Podcast</title>
				<link>https://fblive.yourpodcast.agency/series/football-live/</link>
			</image>
		<itunes:category text="Sports">
			<itunes:category text="Soccer"></itunes:category>
		</itunes:category>
		<googleplay:author><![CDATA[Football Live]]></googleplay:author>
			<googleplay:email>buono@buono.net</googleplay:email>			<googleplay:description>Football Live is exclusive to Brila FM, providing top class, live coverage of the major leagues in Europe to the people of Nigeria. Our show is regularly listened to by audiences on several million in Nigeria but it is available globally for you to listen to live.</googleplay:description>
			<googleplay:explicit>No</googleplay:explicit>
			<googleplay:image href="[http://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg](view-source:http://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg)"></googleplay:image>
			<podcast:locked owner="buono@buono.net">yes</podcast:locked>
		<podcast:guid>96d445a4-b27f-5524-b1fd-52ef55be2a5c</podcast:guid>
		
		
		<generator>https://wordpress.org/?v=6.5.2</generator>

<item>
	<title>Football Live Podcast 20/21 April 2024</title>
	<link>https://fblive.yourpodcast.agency/podcast/football-live-podcast-20-21-april-2024/</link>
	<pubDate>Thu, 25 Apr 2024 16:57:27 +0000</pubDate>
	<dc:creator><![CDATA[Football Live]]></dc:creator>
	<guid isPermaLink="false">https://fblive.yourpodcast.agency/?post_type=podcast&#038;p=17</guid>
	<description><![CDATA[A big weekend in the Premier League as well as two FA Cup semi-final ties at Wembley Stadium across our two live matches. Here are the highlights from our NINE hours of live broadcasts hosted by Richard Parr and Chris Baldwin. The live show is exclusive to brila FM and sponsored by Nairabet.]]></description>
	<itunes:subtitle><![CDATA[A big weekend in the Premier League as well as two FA Cup semi-final ties at Wembley Stadium across our two live matches. Here are the highlights from our NINE hours of live broadcasts hosted by Richard Parr and Chris Baldwin. The live show is exclusive ]]></itunes:subtitle>
	<content:encoded><![CDATA[A big weekend in the Premier League as well as two FA Cup semi-final ties at Wembley Stadium across our two live matches. Here are the highlights from our NINE hours of live broadcasts hosted by Richard Parr and Chris Baldwin. The live show is exclusive to brila FM and sponsored by Nairabet.]]></content:encoded>
	<enclosure url="https://op3.dev/e/fblive.yourpodcast.agency/podcast-download/17/football-live-podcast-20-21-april-2024.mp3" length="11425455" type="audio/mpeg"></enclosure>
	<itunes:summary><![CDATA[A big weekend in the Premier League as well as two FA Cup semi-final ties at Wembley Stadium across our two live matches. Here are the highlights from our NINE hours of live broadcasts hosted by Richard Parr and Chris Baldwin. The live show is exclusive to brila FM and sponsored by Nairabet.]]></itunes:summary>
	<itunes:image href="[https://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg](view-source:https://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg)"></itunes:image>
	<image>
		<url>https://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg</url>
		<title>Football Live Podcast 20/21 April 2024</title>
	</image>
	<itunes:explicit>false</itunes:explicit>
	<itunes:block>no</itunes:block>
	<itunes:duration>9:13</itunes:duration>
	<itunes:author><![CDATA[Football Live]]></itunes:author>	<googleplay:description><![CDATA[A big weekend in the Premier League as well as two FA Cup semi-final ties at Wembley Stadium across our two live matches. Here are the highlights from our NINE hours of live broadcasts hosted by Richard Parr and Chris Baldwin. The live show is exclusive to brila FM and sponsored by Nairabet.]]></googleplay:description>
	<googleplay:image href="[https://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg](view-source:https://fblive.yourpodcast.agency/wp-content/uploads/2024/04/channels4_profile_upscayled_4x.jpg)"></googleplay:image>
	<googleplay:explicit>No</googleplay:explicit>
	<googleplay:block>no</googleplay:block>
</item>
	</channel>
</rss>`





Could be some caching going on??

**Reply by johnspurlock-skymethod** [reply]
dunno, just hit it from another machine, and no items either

is it a cms thing, marked as private or something?  ie try it in a private browser tab

**Comment by britaliano** [comment]
Different browser gave me the same results as you. I turned off nginx on the host, refreshed and now get the episode.

But still the OP3 error.

Thanks for your help so far.

**Reply by johnspurlock-skymethod** [reply]
Ok, feed looks good

Now you may want to disable any "bot mode" or captcha challenges for the feed url itself, you want to make it as available as possible to the hundreds of podcast app/service pollers that will be hitting it.

Looks like requests from Cloudflare (where OP3 runs) are getting challenges:
```
  status: 202,
  headers: [
    [ "cache-control", "no-store,no-cache,max-age=0" ],
    [ "content-type", "text/html" ],
    [ "date", "Thu, 25 Apr 2024 18:51:31 GMT" ],
    [ "expires", "Thu, 01 Jan 1970 00:00:01 GMT" ],
    [ "host-header", "8441280b0c35cbc1147f8ba998a563a7" ],
    [ "set-cookie", "nevercache-b39818=Y;Max-Age=-1" ],
    [ "sg-captcha", "challenge" ],
    [ "transfer-encoding", "chunked" ],
    [ "x-proxy-cache-info", "DT:1" ],
    [ "x-robots-tag", "noindex" ]
  ],
```

**Comment by britaliano** [comment]
Thank you John! That seems to be applied at the host level. I will raise a ticket to look into that.

**Comment by britaliano** [comment]
The host has added op3.dev to the whitelist and the error is gone!

Thank you for your wonderful support, John!

**Reply by johnspurlock-skymethod** [reply]
hey that's great - welcome aboard
